.github/workflows/docs.yml

```diff
@@ -30,3 +30,7 @@ jobs:
             echo "::error::The doc generation produces the following changes:"
             git diff --color --exit-code
           fi
+
+      - name: Validate docs
+        run: |
+          make lintdoc
```

Comment:
I don't have a good technical reason, I just like this better lol

```suggestion
        run: make lintdoc
```

Reply:
sure, why not. In case it becomes two lines in the future we can change again.